docs/linting/troubleshooting/Performance.mdx

```diff
@@ -85,6 +85,8 @@ We recommend you do not use the following rules, as TypeScript provides the same
 - `import/namespace`
 - `import/default`
 - `import/no-named-as-default-member`
+- `import/no-unresolved`
```

Comment:
[Question] I don't think TypeScript enforces this? I added a `require('./nope')` in a new `.ts` file and it didn't complain.

We do have the [`no-var-requires` rule](https://typescript-eslint.io/rules/no-var-requires) though, so maybe it can be noted that this rule is only unnecessary if you ban `require()`?

Reply:
> I don't think TypeScript enforces this? I added a require('./nope') in a new .ts file and it didn't complain.

oh, then maybe it only enforces it for `import` statements and not `require` statements?
with that said, i doubt anyone reading these docs are using require statements in combination with typescript?

Reply:
Correct - a plain require call is completely unchecked. TS only checks a require if it's in the form `import foo = require('foo');`

Reply:
> i doubt anyone reading these docs is using require statements in combination with TypeScript

If that were the case, we wouldn't need the `no-var-requires` rule. But it still exists because it's a valid way to do things in TypeScript (with a caveat around the `any` type).

Reply:
right, so maybe in the docs it should say:

```
- `import/no-unresolved` (as long as you are using `import` over `require`)
```

?

Reply:
👍 sounds good to me. Linking to `no-var-requires` would be helpful too.